src/platform/plugins/shared/data/public/search/session/sessions_mgmt/components/table/utils/map_to_ui_session.ts

```diff
@@ -59,7 +59,11 @@ export const mapToUISession = ({
   if (initialState) delete initialState.searchSessionId;
   // derive the URL and add it in
   const reloadUrl = getUrlFromState(locators, locatorId, initialState);
-  const restoreUrl = getUrlFromState(locators, locatorId, restoreState);
+  const restoreUrl = getUrlFromState(locators, locatorId, {
+    ...restoreState,
+    tabId: 'new',
+    tabLabel: name,
```

Comment:
Do we need to check for tabs or background search feature flags before requesting opening in a new tab?